### ISSUE TITLE
1.16.5 Ice Dragon crash

### ISSUE THREAD

**Issue by SLZT** [Alex-the-666/Ice_and_Fire#4235]
I'm playing with a few friends of mine on a server, but always when we encounter an ice dragon the server crashes due to this error:

```
java.lang.NullPointerException: Ticking entity
	at com.github.alexthe666.citadel.server.entity.datatracker.EntityProperties.getEntity(EntityProperties.java:63) ~[citadel:1.7.2-1.16.5] {re:classloading}
	at com.github.alexthe666.iceandfire.entity.props.FrozenEntityProperties.setFrozenFor(FrozenEntityProperties.java:48) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
	at com.github.alexthe666.iceandfire.entity.IafDragonDestructionManager.lambda$destroyAreaIce$5(IafDragonDestructionManager.java:172) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655) ~[?:?] {}
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[?:?] {}
	at com.github.alexthe666.iceandfire.entity.IafDragonDestructionManager.destroyAreaIce(IafDragonDestructionManager.java:167) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
	at com.github.alexthe666.iceandfire.entity.EntityIceDragon.stimulateFire(EntityIceDragon.java:481) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
	at com.github.alexthe666.iceandfire.entity.EntityDragonBase.tryScorchTarget(EntityDragonBase.java:2175) ~[iceandfire:2.1.8-1.16.5] {re:classloading,pl:runtimedistcleaner:A}
	at com.github.alexthe666.iceandfire.entity.IafDragonFlightManager.update(IafDragonFlightManager.java:78) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
	at com.github.alexthe666.iceandfire.entity.EntityDragonBase.func_70071_h_(EntityDragonBase.java:1619) ~[iceandfire:2.1.8-1.16.5] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_217479_a(ServerWorld.java:611) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.world.World.func_217390_a(World.java:554) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:404) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixin.terraforged.json:common.MixinMinecraftServer,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixin.terraforged.json:common.MixinMinecraftServer,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixin.terraforged.json:common.MixinMinecraftServer,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixin.terraforged.json:common.MixinMinecraftServer,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:829) [?:?] {}
```

**Comment by FlameyFaucet** [comment]
Update. This was fixed with I&F 2.1.9 and Citadel 1.8.1

**Comment by SLZT** [comment]
> Update. This was fixed with I&F 2.1.9 and Citadel 1.8.1

Thanks!
